app/views/trip_families/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="container">
+  <%= simple_form_for([@trip, @trip_family]) do |f| %>
```

Comment:
Can use the helper method here: `[trip, @trip_family]`

Reply:
Done in sha `d57af14`